waspc/src/Wasp/Project/Analyze.hs

```diff
@@ -245,6 +250,14 @@ constructAppSpec waspDir options externalConfigs parsedPrismaSchema decls = do
 
   return $ runValidation ASV.validateAppSpec appSpec
 
+waspDirExists :: Path' Abs (Dir WaspProjectDir) -> IO (Either String (Path' Abs (Dir WaspProjectDir)))
+waspDirExists waspDir = do
+  let waspDotWaspPath = waspDir </> [relfile|.wasp|]
+  isFile <- IOUtil.doesFileExist waspDotWaspPath
+  if isFile
+    then return $ Left "The path to the Wasp project is a file, but it should be a directory."
```

Comment:
The "Wasp project" is the user's project, not the stuff in the `.wasp` directory.

Also, users don't really care about the `.wasp` directory, so this message gives out unnecessary information.  If I read it, I'd think Wasp expects me to create a `.wasp` directory (which isn't true).

We should tell users something that conveys the message "Hey, your `.wasp` file can't be called `.wasp`, it needs to have a proper name," no need to mention the `.wasp` directory at all.

Reply:
Ok, I've re-phrased the message as: *Invalid file name for the .wasp file. Please rename it to [something].wasp.*. What do you think?